tests/io/hdf5_test.py

```diff
@@ -18,7 +18,7 @@ def roundtrip(obj):
 
 def check_roundtrip(obj):
     result = roundtrip(obj)
-    assert sc.identical(result, obj)
+    assert sc.identical(result, obj, equal_nan=True)
```

Comment:
There are no NANs in these tests? Is this used somewhere with hypothesis?

Reply:
This is a leftover from trying hypothesis with this test. Forgot to revert.